src/ChangeStream.php

```diff
@@ -55,14 +57,17 @@ public function getId()
 
     public function key()
     {
-
+        return $this->key;
```

Comment:
Since we're implementing PHP's Iterator interface, we still have to honor the return value contract from [`Iterator::key()`](http://php.net/manual/en/iterator.key.php):

> Returns scalar on success, or NULL on failure.

As-is, I think this always returns an integer. We should probably consult `$this->valid()` first.